src/app.py

```diff
@@ -45,14 +45,9 @@ def form():
         # Get keys, or return None if they don't exist
         start = request.form.get('start')
         end = request.form.get('end')
-        midpoints = request.form.get('midpoints')
+        midpoints = request.form.get('midpoints').split(',')
```

Comment:
this .... this is such a better way of doing this yea idk why i did that earlier